.circleci/config.yml

```diff
@@ -58,12 +61,10 @@ commands:
     description: 'Building and archiving extension artifacts.'
     steps:
       - checkout
-      - restore_dependency_cache
       - install
       - compile
       - package-extensions
       - stage-and-store-artifacts
-      - save_dependency_cache
```

Comment:
for linux we are only caching the top level node_modules dir.  This is providing very little value for this particular job so I just removed it.